packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -98,8 +99,9 @@ const _renderCards = (cards, containerRef) => (
                 src: ArrowRight20,
               },
             }}
-            target={card.cta.target}
             type="link"
+            role="region"
+            aria-labelledby={`region${card.heading}`}
```

Comment:
If we're moving forward with this is this something that's going to continue to be a problem, and should it actually be solved at the `Card` sub-pattern level?